x-pack/test/functional/apps/code/repo_archiver.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import extractZip from 'extract-zip';
+import fs from 'fs';
+import mkdirp from 'mkdirp';
+import path from 'path';
+import rimraf from 'rimraf';
+
```

Comment:
I'd usually define `asyncExtractZip` and `asyncMkdirp` up here like:

```
import { promisify } from 'util';

import extractZip from 'extract-zip';
import mkdirp from 'mkdirp';

const asyncExtractZip = promisify(extractZip);
const asyncMkdirp = promisify(mkdirp);
```

Reply:
done